packages/popover/components/Popover.tsx

```diff
@@ -151,7 +151,7 @@ const Popover: React.StatelessComponent<PopoverProps> = ({
         overlayRoot={overlayRoot}
       >
         {React.isValidElement(trigger) ? (
-          React.cloneElement(trigger, {
+          React.cloneElement(trigger as React.ReactElement, {
```

Comment:
I'm not so sure about the internal react types, but would it also achieve the same if we set the type of trigger to `React.ReactElement` at the props level of this component? 
I think `ReactElement`?

Reply:
This produced the same error. I guess it sees the trigger as any? Some more info: https://stackoverflow.com/a/42261933/3058839